shub/image/test.py

```diff
@@ -105,4 +109,5 @@ def _run_docker_command(client, image_name, command):
                        stderr=True if statuscode else False,
                        stream=False, timestamps=False)
     utils.debug_log("{} results:\n{}".format(command, logs))
+    client.remove_container(container)
```

Comment:
@chekunkov Mm, it doesn't work for me, and looks like [not supported](http://docker-py.readthedocs.io/en/1.10.0/hostconfig/) - author of `docker-py` [recommends](https://github.com/docker/docker-py/issues/1163) using `create-wait-remove` chain instead everywhere.

By the way, TIL `--rm` logic was implemented on client side initially and only now moved to server side in [latest docker](https://github.com/docker/docker/pull/20848).